_caseStudies/2023-05-10-hcd-principles-in-conducting-a-needs-assessment.md

```diff
@@ -0,0 +1,140 @@
+
+A session parallel to the fun exercise was later done in the computer lab, on the same day, where  teaching staff had to create a new account on a blogging site to start a personal blog/online profile, and write short blog posts introducing themselves through a brief biography. This session was more challenging for the teaching staff than the visiting lecturers had anticipated, and shed light on where to begin with the e-learning course.
+
+At this stage, the author finally felt she had enough knowledge to run a survey and ask more informed questions about different kinds of tools they may recognize, the scope of usage, student engagement with chosen tools, and their own chosen alternatives to difficult learning leaps in e-teaching methods. To elicit all this, she ran a survey administered through Google forms as a third and final data-collection exercise to determine the details of what we would spend the second week of our time together learning. After setting everyone up with a gmail account, the 30-question survey took only an hour to complete together, with support where needed.
+
+The conducting of a needs assessment that the author had thought would only take a day, had taken an entire week. The week focused on getting oriented to the teaching staff’s ways of engagement and eliciting their requirements for the e-learning course, by employing various real-time evaluation techniques such as impromptu interviews, focus groups and constant observations. The observations from each day informed the course of the next, and the evaluation continued until there was sufficient information to design the required learning materials. Patricia Rogers (Better Evaluation, 2020) aptly describes real-time evaluation as a combination of the following factors: real-time data, real-time reporting, multiple cycles, double loop learning and diverse sensemaking.
+
+![](/assets/img/posts/RealTimeEvaluations_PRogers_v2.png)
```

Comment:
@mutsacc 
Please check image name to match with what is linked on the site.
`assets/img/posts/RealTime Evaluations_PRogers_v2.jpg`